docs/video/01-build/08-Develop-Managing-Media-Stream.mdx

```diff
@@ -36,9 +36,11 @@ To enable/disable local media, we'll call the `media()` method on the `aliceVide
 
 ## How to know the current status of local audio/video?
```

Comment:
Add playground like how we have in chat and notifs. Also the react playground in get started should contain playground link on all three sections - react-playground.push.org and within the react playground tutorial as well